rclpy/rclpy/qos.py

```diff
@@ -471,3 +471,21 @@ def short_keys(cls):
     def get_from_short_key(cls, name):
         """Retrieve a policy type from a short name, case-insensitive."""
         return cls[name.upper()].value
+
+
+class QoSCompatibility(IntEnum):
+    OK = 0
+    WARNING = 1
+    ERROR = 2
```

Comment:
I think it would be a good idea to create the enum with pybind11 so that the RCL constants aren't duplicated in python

See `ClockType` as an example

https://github.com/ros2/rclpy/blob/4d76c3a522d1ffdffcb2a5e7d925781f01fbfa07/rclpy/src/rclpy/_rclpy_pybind11.cpp#L38-L42
https://github.com/ros2/rclpy/blob/4d76c3a522d1ffdffcb2a5e7d925781f01fbfa07/rclpy/rclpy/clock.py#L21